deploy/docker/file-server.Dockerfile

```diff
@@ -3,42 +3,42 @@ FROM --platform=linux/amd64 docker.io/node:18.12.1-alpine3.17 AS client-base
 
 RUN apk add git --no-cache
 
-# polis-client-admin
-FROM client-base AS client-admin
+# polis-client-participation
+FROM client-base AS client-participation
 
-WORKDIR /client-admin/app
+WORKDIR /app/client-participation/
```

Comment:
`participation` must be built in the same image, or you get an error when loading admin since `admin` is pulling a source file directly from `participation`